Firmware/mmu2.cpp

```diff
@@ -332,7 +334,7 @@ bool MMU2::VerifyFilamentEnteredPTFE()
     } else {
         // else, happy printing! :)
         // Revert the movements
-        current_position[E_AXIS] += logic.ExtraLoadDistance();
+        current_position[E_AXIS] -= MMU2_EXTRUDER_PTFE_LENGTH + MMU2_EXTRUDER_HEATBREAK_LENGTH;
```

Comment:
Same, needs to be adjusted by ExtraLoadDistance

Reply:
Added here 031d00f. Also added brackets around the subtracted constant so the signs are correct.